indico/modules/users/forms.py

```diff
@@ -70,6 +70,16 @@ class UserPreferencesForm(IndicoForm):
         widget=SwitchWidget(),
         description=_('The previewer is used by default for image and text files, but not for PDF files.'))
 
+    add_ical_alerts = StringField(
+        _('Add alerts to iCal'),
+        description=_('Add an alert to exported iCal files/URLs. ' +
+                      'Value should be number of minutes to notify before an event. ' +
```

Comment:
Remove the `+` - strings can be concatenated without it, and i18n extraction will most likely fail with the `+` since that makes it a runtime operation (while `'foo' 'bar'` is combined at parse time and thus not affecting the AST)

Reply:
Ah, OK, got a pre-commit complaint about the line being too long. I didn't realize that about string concatenation